app/views/admin/site_customization/pages/index.html.erb

```diff
@@ -11,24 +11,22 @@
   <table class="cms-page-list">
     <thead>
       <tr>
-        <th><%= t("admin.site_customization.pages.page.title") %></th>
+        <th><%= t("admin.site_customization.pages.page.slug") %></th>
         <th><%= t("admin.site_customization.pages.page.created_at") %></th>
         <th><%= t("admin.site_customization.pages.page.updated_at") %></th>
         <th><%= t("admin.site_customization.pages.page.status") %></th>
-        <th><%= t("admin.site_customization.pages.page.locale") %></th>
         <th class="small-4"><%= t("admin.actions.actions") %></th>
       </tr>
     </thead>
     <tbody>
     <% @pages.each do |page| %>
       <tr id="<%= dom_id(page) %>">
         <td>
-          <%= link_to page.title, edit_admin_site_customization_page_path(page) %>
+          <%= link_to page.slug, edit_admin_site_customization_page_path(page) %>
```

Comment:
In all other translatable sections we show the title. @decabeza @papayalabs  Which one do you guys think is more convenient here?